src/Resources/contao/templates/backend/be_main.html5

```diff
@@ -94,7 +94,7 @@
 
   <?php if ($this->isPopup): ?>
     <script>
-      if ($$('.tl_tree_checkbox,.tl_tree_radio').length > 0) {
+      if ($$('.tl_tree_checkbox,.tl_tree_radio').length > 0 && !document.location.search.test('act=select')) {
```

Comment:
That would mean the buttons are not disabled when in select mode, which is not what we want, right?